### ISSUE TITLE
one char not rendering after patching

### ISSUE THREAD

**Issue by rodhash** [rbong/flog-symbols#2]
Hi

After patching my font it's working great except for one single char, the top char:

<img width="297" alt="image" src="https://github.com/user-attachments/assets/f4851004-d974-45c9-aa8b-cdb4f03f8670">

All other seem to be rendering just fine:

<img width="210" alt="image" src="https://github.com/user-attachments/assets/7777ceb9-ab8a-4828-8e2e-fe5da49626d1">

I know this font is not meant to be installed but patched.. but if I install this font and use it as a 2nd font then 1st char renders ok:

iTerm2 setting:
<img width="252" alt="image" src="https://github.com/user-attachments/assets/f3294232-e4bd-4536-abce-0507644107ac">

Then:
<img width="244" alt="image" src="https://github.com/user-attachments/assets/a5585d2f-1d0b-4373-a3ed-0b41c47c9339">

Any idea?

**Comment by rodhash** [comment]
I forgot to mention, in the screenshots above I didn't actually patched but merged both fonts using fontforge.

Now I tried patching my current font adding this flog-font as custom. 1st char now works but it break others:

first char:
<img width="289" alt="image" src="https://github.com/user-attachments/assets/91856094-9583-438d-b573-2a682a988303">

others:
<img width="251" alt="image" src="https://github.com/user-attachments/assets/aa2c38ba-6265-4a67-8d2e-b253351a6dd9">


ps: I know kitty has this working fine but I'm avoiding using it because it's not allowed in my work computer.

**Comment by rbong** [comment]
Help very much appreciated getting the font symbols to line up. Actually the fact that I [couldn't get fonts to line up and the author of Kitty saying there is no universal fix](https://github.com/kovidgoyal/kitty/discussions/7680) led me to take the path of patching terminals instead.

However I believe that it may possible to have something better than what we have now. I know that the original screenshots for [gitgraph.nvim](https://github.com/isakbm/gitgraph.nvim) were using font patching, and those lined up perfectly. I have not been able to get my symbols to match up perfectly in any one font size on Kitty, so this is at least one better. I asked the author of gitgraph.nvim for help in a related Reddit thread, and also they said they would post a font patching guide. Nothing yet though.

The methodology right now is that I copied the original branch drawing symbols from Nerd Fonts and modified them. Which you would think would work - but doesn't. I believe most terminals have special alignment for this character range or, like Kitty, they overwrite this character range. Perhaps this is how gitgraph.nvim got these characters to line up, because IIRC they patched over the original box drawing symbol range instead of defining a new range of symbols in the private use area of the font.

tl;dr - Help needed, may be possible to improve, may not. Font patching will likely never be perfect.

Please share your terminal and font settings if you can. Thanks

**Comment by rodhash** [comment]
I've tried different patching and merging methods and also messed with the glyph scaling/shifting but I always get some chars not properly aligned.

I'd be happy to help if I can, so far the only workaround I found was to use my own font (as usual) and then set this flog font to render only this particular range of chars.

In my iTerm2 config:

<img width="423" alt="image" src="https://github.com/user-attachments/assets/857c0399-b6e9-4221-8762-891d5a8676e4">

Now I see all my previous font glyphs fine + flog glyphs properly aligned (even that first one going rogue)

About this:
> Please share your terminal and font settings if you can. Thanks

iTerm2: `Build 3.5.8`

Fonts used:
<img width="247" alt="image" src="https://github.com/user-attachments/assets/5c88c4af-22e1-4380-9223-ad6d4311dcae">
